src/tests/simulator/cell.test.js

```diff
@@ -0,0 +1,40 @@
+
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { expect } from 'chai';
+
+import Cell from '../../containers/simulator/cell';
+
+it('renders without crashing', () => {
```

Comment:
Do you not need a `describe` call before the `it`? e.g.

```
describe("Filter", () => {
    it("Does not modify tokens if no empty lines or END found", () => {
```

Reply:
Nope, it's optional kind of like a namespace.
The base test setup is from the create-react-app (CRA) template but I've brought in enzyme from Airbnb which is awesome

Reply:
I thought the `describe` call added the name of the test suite to each test so in my example, the test runner would output `Filter Does not modify tokens if....`.  Is it easy to tell which test file is failing without it?